lib/Graph/Graph.cpp

```diff
@@ -1434,6 +1433,14 @@ DequantizeNode *Function::createDequantize(llvm::StringRef name,
   return addNode(new DequantizeNode(name, outTy, input));
 }
 
+DequantizeNode *Function::createDequantize(llvm::StringRef name,
+                                           NodeValue input, TypeRef outTy) {
```

Comment:
Could we have the other createDequantize call this one, so that we don't duplicate the assert?
Ditto for createQuantize.